apps/spark/src/main/java/com/linkedin/openhouse/jobs/scheduler/JobsScheduler.java

```diff
@@ -350,14 +358,16 @@ protected static TablesClientFactory getTablesClientFactory(CommandLine cmdLine)
         DatabaseTableFilter.of(
             cmdLine.getOptionValue("databaseFilter", ".*"),
             cmdLine.getOptionValue("tableFilter", ".*"));
+    CreationTimeFilter timeFilter =
+        CreationTimeFilter.of(Integer.parseInt(cmdLine.getOptionValue("timeFilter", "1")));
```

Comment:
Should we add creationTime field to the existing DatabaseTableFilter? Pro: less code change amplification.

Reply:
Added interface and new filter Class to keep it cleaner. In future there maybe more filter criteria requirements and putting all in DatabaseTable filter will make it tightly scoped. Open to discussion if you feel strongly about it.

Reply:
I'd recommend having one filter class that we construct here, we should keep code change amp low.

Reply:
SG, updated the implementation.